docs/deployment/koyeb.md

```diff
@@ -0,0 +1,50 @@
+---
+id: koyeb
+title: Deploying with Koyeb
+sidebar_label: Koyeb
+description: How to deploy Backstage to Koyeb
+---
+
+This guide explains how to deploy Backstage to [Koyeb](https://www.koyeb.com/), a serverless platform that provides the fastest way to deploy applications globally. Koyeb supports git-driven and container-based deployments.
+
+Before you begin, make sure you have a [Koyeb account](https://app.koyeb.com/) to follow this guide.
+
+## Configuring the CLI
+
+First, install the
+[Koyeb CLI](https://www.koyeb.com/docs/quickstart/koyeb-cli) and follow the instructions in the [quickstart guide](https://www.koyeb.com/docs/quickstart/koyeb-cli) to login.
+
+Then, configure your `app-config.yaml` with your `baseURL`:
+
+```yaml
+app:
+  # Should be the same as backend.baseUrl when using the `app-backend` plugin
+  baseUrl: https://<your-app>.koyeb.app
+
+backend:
+  baseUrl: https://<your-app>.koyeb.app
+  listen:
+    port:
+      $env: PORT
```

Comment:
```suggestion
    port: ${PORT]
```

this shorthand is much more convenient and we're trying to lean toward it where possible

Reply:
Noted 👍 thanks for pointing this out! I changed it with my last commit which incorporated all of your feedback.